### DISCUSSION TITLE
Tailwind class sorting in templ with prettier

### DISCUSSION THREAD

**Discussion by llamux** [a-h/templ#681] [Show and tell]
I wanted this in templ: https://tailwindcss.com/blog/automatic-class-sorting-with-prettier#how-classes-are-sorted

So I made this: https://github.com/llamux/prettier-plugin-tailwindcss-extra

**Comment by joerdav** [comment]
Very cool!

**Comment by redhotvengeance** [comment]
@llamux This is exactly what I was looking for. Thanks!

**Comment by joerdav** [comment]
Will close this discussion and keep it for people searching in the future.
